_rules/presentational-children-no-focusable-content-307n5z.md

```diff
@@ -127,16 +136,93 @@ This element with `menuitemcheckbox` role has a checkbox as a child. Because the
-<a href="https://w3.org">W3C Website</a>
+<table>
+		<thead>
+				<tr>
+						<th colspan="2">semantic table header</th>
+				</tr>
+		</thead>
+		<tbody>
+				<tr>
+						<td><button>button in semantic table - no problem</button></td>
+						<td><a href="#">anchor in semantic table - no problem</a></td>
+				</tr>
+		</tbody>
+</table>
 ```
 
+#### Inapplicable Example 2
+
+This `<a>` element has a `link` role, which does not have [presentational children][].  So this element - perhaps surprisingly - does not fail this rule, because it's inapplicable.  This is the case even though it's invalid HTML.  Any time there's a link as the <i>outer</i> element, that will be inapplicable under this rule.  (If the elements were switched - that is, if the `<button>` element contained the `<a>` element - then that would fail this rule.)
+
+```html
+<a href="https://w3.org"><button>W3C Website</button></a>
+```
+
+#### Inapplicable Example 3
+
+The applicability of this rule does not include elements with an <i>explicit</i> role of `presentation`.  Only <i>implicit</i> (through roles with [presentational children][]).  This example shows that. 
+
+```html
+<ul>
+	<li role="presentation">
+		<a href="#">inapplicable</a>
+	</li>
+</ul>
+```
+
+#### Inapplicable Example 4
+
+The applicability of this rule does not include hidden elements.  It only includes <i>presentational</i> elements (through roles with [presentational children][]).  These two terms - "hidden" and "presentational" - are often confused.  When an element is "hidden", that means that it is completely omitted from the accessibility tree.  When an element is "presentational", that means that its semantic role is omitted from the accessibility tree, but its content remains.
+
+The [ARIA 1.2 specification]https://www.w3.org/TR/wai-aria-1.2/#presentation) mentions this confusion:
+<ul>
+<li>
+<q>Many individuals erroneously consider role="presentation" to be synonymous with aria-hidden="true"</q>
+</li>
+<li>
+<q>... the presentation role causes a given element to be treated as having no role or to be removed from the accessibility tree, but does not cause the content contained within the element to be removed from the accessibility tree.</q>
+</li>
+</ul>
+
+Adding to this confusion is a third term: "decorative".  The words "decorative" and "presentational" are often used interchangeably, but that usage is inaccurate.  The word "decorative" often appears in a sentence such as "marking an image as decorative" - that is, by adding `alt=""` to an `<img>` element.  "Decorative" in that context <i>does</i> mean "hidden" - and "hidden", again, is different from "presentational" - so using "decorative" and "presentational" interchangeably is inaccurate.  At the time of writing (August 2023), the ACT definition of "[marked as decorative][]" unfortunately encourages this inaccurate usage: it states that <q>An element is marked as decorative if ... it has an explicit role of none or presentation</q>.
```

Comment:
That's why we use "**marked** as decorative", and not "decorative" 😄 
Being "marked as decorative" is showing author **intention** to somehow remove the content. It doesn't necessarily means that the content will effectively be removed/hidden/… (because the presentational role conflict resolution can trigger).

I also consider "decorative" as being a semantic property of the page. "presentational" (or role of `presentation` (post-conflict resolution)) is more of a functional/effective property of the page. The same way that "being a label (WCAG sense of the term)" is a semantic property while using a `label` element is the functional property (and the way to convey  that semantics programmatically). That is, an image is "decorative" not because it has `alt=""` but because it is "serving only an aesthetic purpose, providing no information, and having no functionality" (WCAG's "pure decoration"). Adding `alt=""` is a way to convey programmatically the fact that the image is decorative, so that ATs can treat it differently from a non-decorative image.

In that spirit, "marked as decorative" is "the author tried to convey programmatically the semantic information that something is decorative". This may work as intended (and the stuff will be either presentational or hidden), but this may also fail (and the stuff will be exposed as normal).

Reply:
Okay, so decorative != marked as decorative.  Even if that's okay (which is unclear to me), I still think we have a problem.  

I would draw a venn diagram here if I could, but I can't, so here is a tree which represents a venn diagram of the situation you described.  (In this venn diagram tree, all child sets totally consume the parent set.  eg. Set B minus Set C minus Set D = an empty set.)

- Set A: marked as decorative 
  - Set B: "marked as decorative" worked 
    - Set C: presentational 
    - Set D: hidden
  - Set E: "marked as decorative" did not work (==> still has semantics) 

That presents two problems:
1) Sets C and D are so different, I think it's confusing for us to include them both under "Set B: "marked as decorative" worked".
2) I think that "Set D: hidden" corresponds to what most people think when they hear "marked as decorative".  Also, I suspect that it overlaps precisely with their ideas of "decorative" and WCAG's "pure decoration" - and that's a good thing.  If we go with the the above venn diagram, someone might say that ACT is out of touch, and they would be right.

Reply:
While I think this feedback about how some definitions can be perceived is important, I don't think this is a good place to discuss it. I would suggest @dan-tripp-siteimprove opens an issue to discuss these points, and rewrite the example description is a less verbose manner.